package.json

```diff
@@ -109,6 +109,10 @@
       "eslint --fix",
```

Comment:
While testing this PR I found out that there's some conflicts with the rules between `prettier` and `eslint` for JS. I.e., `prettier` will add spaces after function definitions and then `eslint` will remove it right after. We might want to consider adjusting our rules in the future
https://stackoverflow.com/questions/44690308/whats-the-difference-between-prettier-eslint-eslint-plugin-prettier-and-eslint